lib/public/views/Runs/ActiveColumns/runsPerPeriodActiveColumns.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+import runNumberFilter from '../../../components/Filters/RunsFilter/runNumber.js';
+import { displayRunNumber } from '../format/displayRunNumber.js';
+import { formatRunStart } from '../format/formatRunStart.js';
+import { formatRunEnd } from '../format/formatRunEnd.js';
+import { formatTimestamp } from '../../../utilities/formatting/formatTimestamp.js';
+import o2endFilter from '../../../components/Filters/RunsFilter/o2stop.js';
+import o2startFilter from '../../../components/Filters/RunsFilter/o2start.js';
+import fillNumbersFilter from '../../../components/Filters/RunsFilter/fillNumbers.js';
+import { frontLink } from '../../../components/common/navigation/frontLink.js';
+import { profiles } from '../../../components/common/table/profiles.js';
+import { formatDetectorQuality } from '../format/formatDetectorQuality.js';
+import { formatAliceCurrent } from '../format/formatAliceCurrent.js';
+
+export const RunsPeriodActiveColumns = {
```

Comment:
If you are displaying run, please use the runs active columns and profiles.
Here, this looks a lot like runs overview active columns, with its (quite numerous) defaults

Reply:
done